docs/general/ddl/replication.rst

```diff
@@ -1,73 +1,100 @@
+setting.
+
+For example::
 
     cr> create table my_table10 (
     ...   first_column integer,
     ...   second_column text
     ... ) with (number_of_replicas = 0);
     CREATE OK, 1 row affected (... sec)
 
-The ``number_of_replicas`` property also accepts an string as parameter that
-contains a ``range``.
-
-A range is a definition of minimum number of replicas to maximum number of
-replicas depending on the number of nodes in the cluster. The table below shows
-some examples.
-
-===== =========================================================================
-Range Explanation
-===== =========================================================================
-0-1   Will create no replicas if only one node. This will result in a yellow
-      cluster health.
-
-      One replica if more than one node.
------ -------------------------------------------------------------------------
-2-4   Table requires at least two replicas to be fully replicated.
-
-      Will create up to four replicas if nodes are added.
-
-      If you have less than three nodes, one or more replica shards will be
-      located on the same node as the primary shard. This will result in a
-      yellow cluster health.
------ -------------------------------------------------------------------------
-0-all Will expand the number of replicas to the available number of nodes.
-===== =========================================================================
-
-For details of the range syntax refer to :ref:`the CREATE TABLE
-documentation <sql-create-table-number-of-replicas>`.
-
-.. NOTE::
-
-    The number of replicas can be changed at any time.
+As well as configuring a single fixed number of pre-replica shards, you can use
+a string to configure a range by specifying a minimum and a maximum (dependent
+on the number of nodes in the cluster).
+
+Here are some examples of replica ranges:
+
+========= =====================================================================
+Range     Explanation
+========= =====================================================================
+``0-1``   If you only have one node, CrateDB will not create any replicas. If
+          you have more than one node, CreateDB will create one replica per
+          shard.
+
+          This range is the default value.
+--------- ---------------------------------------------------------------------
+``2-4``   Each table will require at least two replicas for CrateDB to consider
+          it fully replicated (i.e., a *green status*).
+
+          If the cluster has five nodes, CrateDB will create four replicas,
+          with each replica located on a different node from its respective
+          primary.
+
+          If a cluster has four or fewer nodes, CreateDB will have to locate
+          one or more replica shards will be on the same node as the respective
+          primary shard. If this happens, your cluster will have a *yellow*
```

Comment:
```suggestion
          If a cluster has four or fewer nodes, CreateDB would have to locate
          one or more replica shards on the same node as the respective
          primary shard. This is not permitted and such your cluster will have a *yellow*
```

Reply:
> because even though it's not permitted, it can still happen

No this cannot happen, its really enforced. CrateDB will never allocate a replica shard on the same node of the primary shard.

Reply:
So we should also remove the "Normally" inside `Normally, CrateDB dynamically allocates shards to...`

Reply:
ah, I see. thanks for the explanation! I have an outstanding question for you that is related to this. see https://github.com/crate/crate/issues/11263

if you have time, could you take a look at that? based on the discussion there, I think I should probably make a follow-up PR to really make this behavior clear in other places too